common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -24,7 +24,7 @@ const val ATTRIBUTES_ERROR_RESPONSE_KEY = "attributes_error_response"
 const val ATTRIBUTE_ERRORS_KEY = "attribute_errors"
 
 /** @suppress */
-internal typealias CustomerInfoCallback = Pair<(CustomerInfo) -> Unit, (PurchasesError) -> Unit>
+internal typealias CustomerInfoCallback = Pair<(CustomerInfo) -> Unit, (PurchasesError, isServerError: Boolean) -> Unit>
```

Comment:
Hmm it's possible but I don't think it's the way to do it since `PurchasesError` is actually a public class, so it would mean that data would be public. This seems to be a very specific error which shouldn't be exposed.

Reply:
Oh gotcha. We solve this in iOS by having an internal error at the `Backend` level, that we don't convert to the public type until the top of the hierarchy.